src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -1472,6 +1474,65 @@ void availableNoticeIsSentUponCheckInWhenRequesterBarcodeWasChanged() {
       getUserContextMatchers(updatedRequesterJson)));
   }
 
+  @Test
+  void linkItemToHoldTLRWhenCheckedIn(){
+    configurationsFixture.deleteTlrFeatureConfig();
```

Comment:
Can we add this to BeforeAll?

Reply:
no, because configurationsFixture is not static, but we can use reconfigureTlrFeature(TlrFeatureStatus.NOT_CONFIGURED) method which deletes all existing tlr configs